scalameta/semantic/shared/src/main/protobuf/semanticdb.proto

```diff
@@ -31,10 +31,9 @@ message Range {
 
 message Message {
   enum Severity {
-    UNKNOWN = 0;
-    INFO = 1;
-    WARNING = 2;
-    ERROR = 3;
+    INFO = 0;
```

Comment:
The enum id should never change, for example by moving INFO from 1 to 0 then when we read an older semanticdb INFO messages become WARNING. It seems you can't reserve an enum id, but that's OK, you can remove the UNKNOWN field if you like and start counting at 1.

iirc, for protobuf enums it's conventional to have an UNKNOWN value with tag 0. This is because enums are implemented as int32 and 0 is the default value for such fields.